### PR TITLE
Update read-key-value-pairs.mdx

### PR DESCRIPTION
**Pull request by thomasgauvin**

### CODE REVIEW

**Comment by thomasgauvin** [comment]
Addresses https://github.com/cloudflare/cloudflare-docs/issues/18226

**Review by Oxyjun — APPROVED** [review]

File: src/content/docs/kv/api/read-key-value-pairs.mdx [hunk 1]
```diff
@@ -58,10 +58,10 @@ The `get()` method returns a promise you can `await` on to get the value. If the
 - `type`: `"text" | "json" | "arrayBuffer" | "stream"`
   - Optional. The type of the value to be returned. `text` is the default.
 - `options`: `{
-    cacheTtl: number,
+    cacheTtl?: number,
     type: "text" | "json" | "arrayBuffer" | "stream"
```
Comment (thomasgauvin) [inline src/content/docs/kv/api/read-key-value-pairs.mdx]:
```suggestion
    type?: "text" | "json" | "arrayBuffer" | "stream"
```

File: src/content/docs/kv/api/read-key-value-pairs.mdx [hunk 2]
```diff
@@ -91,10 +91,10 @@ Metadata is a serializable value you append to each KV entry.
...
```
Comment (thomasgauvin) [inline src/content/docs/kv/api/read-key-value-pairs.mdx]:
```suggestion
    type?: "text" | "json" | "arrayBuffer" | "stream"
```

File: src/content/docs/kv/api/read-key-value-pairs.mdx [hunk 3]
```diff
@@ -46,7 +46,9 @@ The following methods are provided to read from KV:
 To get the value for a given key, call the `get()` method on any KV namespace you have bound to your Worker code:
 
 ```js
-env.NAMESPACE.get(key, type?, options?);
+env.NAMESPACE.get(key, type?);
+//OR
```
Comment (GregBrimble) [inline src/content/docs/kv/api/read-key-value-pairs.mdx]:
```suggestion
// OR
```

File: src/content/docs/kv/api/read-key-value-pairs.mdx [hunk 4]
```diff
@@ -79,7 +81,9 @@ The `get()` method may return stale values. If a given key has recently been rea
 To get the value for a given key along with its metadata, call the `getWithMetadata()` method on any KV namespace you have bound to your Worker code:
 
 ```js
-env.NAMESPACE.getWithMetadata(key, type?, options?);
+env.NAMESPACE.getWithMetadata(key, type?);
+//OR
```
Comment (GregBrimble) [inline src/content/docs/kv/api/read-key-value-pairs.mdx]:
```suggestion
// OR
```

**Review by GregBrimble — APPROVED** [review]
Better than it was! I'll again tag @KianNH to see if he's at all interested in looking into automation here, but let's ship this now as it is :)

**Comment by thomasgauvin** [comment]
> Better than it was! I'll again tag @KianNH to see if he's at all interested in looking into automation here, but let's ship this now as it is :)

Thanks for reporting this @GregBrimble! good catch

**Pull request merged**